lib/karafka/setup/configurators/celluloid.rb

```diff
@@ -3,13 +3,16 @@ module Setup
     class Configurators
       # Class responsible for setting up Celluloid settings
       class Celluloid < Base
+        # How long should we wait for actors (listeners) before forcefully shutting them down
```

Comment:
I think that it is obvious, but we can be precise and write that this time is set in seconds, not in minutes for example.